scss/6-components/_disclosure.scss

```diff
@@ -1,33 +1,44 @@
-/** @define disclosure **/
+// ============================================================================
+// Disclosure
+// ============================================================================
+
+/** @define disclosure; weak **/
 .cads-disclosure {
-  &__details {
+  .cads-disclosure__details {
```

Comment:
Does this change make the selectors more specific? If we don't need the change in specificity, then can we get the classes outside of the `.cads-disclosure` wrapper?

Reply:
Ah yeah it does, happy to move it out. I changed this short hand selectors are a bit of a nightmare for refactoring without expanding things out in your head, but we probably don't want the extra specificity.